### PR TITLE
CSS fixes when resizing library browser

### LINKED ISSUE #946: Tabs header overlaps page title for some languages in TV layout
**Describe the bug**
Tabs header may overlap page title for "long string" languages.

**To Reproduce**
1. Turn on Russian language.
2. Turn TV layout.
3. Go to TV shows.
4. Tabs header overlaps page title.

**Expected behavior**
Maybe tabs header should be scrollable. :thinking: 

**Screenshots**
tabs
_Blue area is `headerTabs`_

**System (please complete the following information):**
 - OS: Kubuntu 19.04 64 bit
 - Browser: Firefox, Chrome
 - Jellyfin Version: 10.5.0

### PR DESCRIPTION
**Pull request by ferferga**
These fixes work much better for pages with a lot of tabs, such as Music and TV Shows, **removing all the text overlapping**. Also, now the scaling of the cards is "synchronised" with the scaling of the header, being more seamless.

**Before**
before

**After**
after

Fixes #946

### CODE REVIEW

**Comment by dmitrylyzo** [comment]
Doesn't work for TV layout in Music library (for example, Russian, Greek, Kasakh and Spanish).
tbh, I am not a fan of tabs in the second row.

**Comment by ferferga** [comment]
@dmitrylyzo Let me check TV. Forgot to reload the page after enabling TV layout.

What you propose for replacing the two rows? I don't dislike them tbh.

**Comment by dmitrylyzo** [comment]
> What you propose for replacing the two rows?

_from #946_
> Maybe tabs header should be scrollable.

**Comment by dkanada** [comment]
It already is scrollable I believe, but it definitely needs to be on the second row for smaller screens.

**Comment by Delgan** [comment]
Isn't the usual solution to group tabs in a menu, if there is not enough space to display all of them?

That's what Github does, for example:
Screenshot_2020-06-19 jellyfin jellyfin-web
Screenshot_2020-06-19 jellyfin jellyfin-web(1)

However, I don't know how difficult it is to implement.

**Comment by dkanada** [comment]
I actually dislike that method because it requires another click to get to the tabs. Since our header hides automatically during scroll it doesn't actually take up space normally, so the second row for tabs is fine in my opinion.

**Comment by ferferga** [comment]
* TV layout tabs are scrollable. If the width is smaller, two rows are displayed, also scrollable.
* Tabs in second row are scrollable now also in desktop layout when width is really small

image

File: src/components/maintabsmanager.js [hunk 1]
```diff
@@ -140,7 +140,7 @@ define(['dom', 'browser', 'events', 'emby-tabs', 'emby-button'], function (dom,
             var index = 0;
 
             var indexAttribute = selectedIndex == null ? '' : (' data-index="' + selectedIndex + '"');
-            var tabsHtml = '<div is="emby-tabs"' + indexAttribute + ' class="tabs-viewmenubar"><div class="emby-tabs-slider" style="white-space:nowrap;">' + getTabsFn().map(function (t) {
+            var tabsHtml = '<div is="emby-tabs"' + indexAttribute + ' class="tabs-viewmenubar"><div class="emby-tabs-slider scrollX" style="white-space:nowrap;">' + getTabsFn().map(function (t) {
```
[the remark this answers is not in the document]
Comment (ferferga) [reply src/components/maintabsmanager.js:143]: @dmitrylyzo I couldn't scroll using the arrow keys of the keyboard when the tabs were focused until I added this, even after checking that the parent element (as you mention) had the class. Are you sure? Can you verify please?

[the remark this answers is not in the document]
Comment (dmitrylyzo) [reply src/components/maintabsmanager.js:143]: I did these 2 changes and it works.

**Comment by ferferga** [comment]
@dmitrylyzo Done, thanks!

**Pull request merged**